src/components/Upgrade/HighlightScreen.js

```diff
@@ -61,7 +61,7 @@ const HighlightScreen = ({
           flex-shrink: 0;
           width: ${verticalMode ? 'auto' : `${RATIO_LEFT * 100}%`};
           height: ${verticalMode ? 'auto' : '100%'};
-          padding: ${verticalMode ? '30px 20px 90px' : '10vh 60px 90px'};
+          padding: ${verticalMode ? '30px 20px 90px' : `10vh 40px 90px`};
```

Comment:
I wasn't sure if using `GU` here was a good choice, we might not want this to change if `GU` changes.